docs/features/search/architecture.md

```diff
@@ -4,8 +4,6 @@ title: Search Architecture
 description: Documentation on Search Architecture
 ---
 
-# Search Architecture
```

Comment:
I removed these, they are duplicate with the frontmatter: 
![image](https://user-images.githubusercontent.com/648527/128311246-8f101088-a65c-4ddc-ad04-1e2d083a7e9b.png)

Reply:
👍